lib/account.js

```diff
@@ -25,7 +25,9 @@ import {
   createFileName,
   getFileName,
   boostsFile,
-  pathToDMs
+  pathToDMs,
```

Comment:
this is not used

Reply:
Not sure what you mean here, wrong line?

Reply:
nevermind i misread the diff.